generators/server/templates/src/main/resources/logback-spring.xml.ejs

```diff
@@ -152,6 +152,10 @@
     <%_ if (cacheProvider === 'infinispan') { _%>
     <logger name="unknown.jul.logger" level="WARN"/>
     <%_ } _%>
+    <!-- See https://github.com/jhipster/generator-jhipster/issues/13835 -->
+    <logger name="Validator" level="INFO"/>
+    <!-- See https://github.com/jhipster/generator-jhipster/issues/14444 -->
+    <logger name="_org.springframework.web.servlet.HandlerMapping.Mappings" level="INFO"/>
```

Comment:
Typo with additional _ ?

Reply:
No in fact not. It's a hidden logger that is prefixed with _

Reply:
May be lets add a comment so no one thinks its a typo 😂